BEST_PRACTICES.md

```diff
@@ -1,46 +1,35 @@
-# Best Practices & Style Guide
+# Best Practices
 
-## Table of Contents
+These are things we've found to be helpful in creating readable, maintainable code rather than hard requirements. Hard requirements live in the [Style Guide](STYLE_GUIDE.md)
+
+## Table of contents
 
 * [Xcode](#xcode)
+    * [Version](#version)
+    * [Project structure](#project-structure)
 * [Versioning](#versioning)
 * [Class structure](#class-structure)
-* [Commented code](#commented-code)
 * [Comments](#comments)
-* [Blocks, delegates and data source](#blocks-delegates-and-data-source)
+* [Extension method naming](#extension-method-naming)
+* [Blocks/closures vs. delegates](#blocks-closures-vs-delegates)
 * [View controllers](#view-controllers)
-* [Assets](#assets)
-* [Property Observing](#property-observing)
+* [Property observing](#property-observing)
 * [Networking](#networking)
-* [View layout](#view-layout)
-* [Optional Force Unwrapping](#optional-force-unwrapping)
-* [Lazy loading](#lazy-loading)
-* [Swift style guide](#swift-style-guide)
-  * [Linting](#linting)
-  * [Naming](#naming)
-    * [Class Prefixes](#class-prefixes)
-  * [Spacing and Indentation](#spacing-and-indentation)
-  * [Classes and Structs](#classes-and-structs)
-    * [Protocol Conformance](#protocol-conformance)
-    * [Computed Properties](#computed-properties)
-    * [Example definition](#example-definition)
-  * [Function Declarations](#function-declarations)
-  * [Closure Expressions](#closure-expressions)
-  * [Types](#types)
-    * [Constants](#constants)
-    * [Optionals](#optionals)
-    * [Struct Initializers](#struct-initializers)
-    * [Type Inference](#type-inference)
-    * [Syntactic Sugar](#syntactic-sugar)
-  * [Semicolons](#semicolons)
-  * [Resources](#resources)
 
 ## Xcode
 
 ### Version
 
 The recommended version of Xcode (and Swift) for all purposes is the current available [in the App Store](https://itunes.apple.com/no/app/xcode/id497799835?mt=12).
 
+Make sure to coordinate with your project team when upgrading to a new version, particularly if a new version of Swift is involved.
+
+### Project structure
+
+The physical files should be kept in sync with the Xcode project files in order to avoid file sprawl. Any Xcode groups created should be reflected by folders in the filesystem. Code should be grouped by type and feature for greater clarity.
+
+Make sure to use an [app-template](https://github.com/bakkenbaeck/app-template) for consistency.
```

Comment:
Are we going to use this? I haven't in a long time.

Reply:
I have not since I've been on the same project since I started. In the past I have found it extremely helpful to have an app template, especially for things like "Setting up a bunch of boilerplate related to CI"

Reply:
We should evaluate whether the current template is up-to-date for what we need there.

Reply:
That would make more sense. Actually, can't we build something that integrates with Xcode anymore? So we can start the template from the `new project` tab?

Reply:
Yes, I fought with this [at a prior job](https://github.com/vokal/Xcode-Template). However, I think that's more of an issue for that repo than for this one. I think we should still point to a common template here.